src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/components/enterprise-alert/enterprise-alert.module.css

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+.root {
+	--tag-margin: 12px;
+
+	background: var(--token-color-surface-faint);
+	border: 1px solid var(--token-color-border-faint);
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	padding: 16px;
+	border-radius: 6px;
+	color: var(--token-color-foreground-primary);
+
+	&.inline {
+		background: none;
+		border: none;
+		padding: 0 7px;
+		display: inline-flex;
+		vertical-align: middle;
+
+		--tag-margin: 0;
+
+		&:first-child {
```

Comment:
This is for table styles

Reply:
🙋🏻 Which tables? I think I'm missing some context!

Reply:
See this example where the alert renders [inline in a table cell](https://dev-portal-git-ksupdate-enterprise-alert-hashicorp.vercel.app/consul/docs/nia/configuration#consul-kv-source-input)